numpy/tests/test_ctypeslib.py

```diff
@@ -9,20 +9,30 @@
 from numpy.testing import assert_, assert_array_equal, assert_raises, assert_equal
 
 try:
+    import ctypes
+except ImportError:
+    ctypes = None
+else:
     cdll = None
+    test_cdll = None
     if hasattr(sys, 'gettotalrefcount'):
         try:
             cdll = load_library('_multiarray_umath_d', np.core._multiarray_umath.__file__)
         except OSError:
             pass
+        try:
+            test_cdll = load_library('_multiarray_tests', np.core._multiarray_tests.__file__)
```

Comment:
Long lines here and above.

Reply:
Missed this one. It's not clear to me why `load_library` is needed here - the tests in cpython just use `ctypes.CDLL(np.core._multiarray_tests.__file__)`, and presumably work just fine.

Reply:
Are you looking to fix this, or are you going to try the cpython approach and remove an argument?

Reply:
Oops, `load_library` is in the public interface, forgot about that.